packages/foreman/rubygem-oauth/rubygem-oauth.spec

```diff
@@ -2,16 +2,16 @@
 %global gem_name oauth
 
 Name: rubygem-%{gem_name}
-Version: 0.5.10
+Version: 1.0.0
 Release: 1%{?dist}
 Summary: OAuth Core Ruby implementation
 License: MIT
 URL: https://github.com/oauth-xx/oauth-ruby
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: ruby >= 2.0
-BuildRequires: ruby >= 2.0
+Requires: ruby >= 2.7
+BuildRequires: ruby >= 2.7
```

Comment:
I think you'll have to package `version_gem` first: https://github.com/oauth-xx/oauth-ruby/blob/main/oauth.gemspec#L6